manager/manager.go

```diff
@@ -140,10 +140,12 @@ func New(memoryCache *memory.InMemoryCache, sysfs sysfs.SysFs, maxHousekeepingIn
 	dockerStatus, err := docker.Status()
 	if err != nil {
 		glog.Warningf("Unable to connect to Docker: %v", err)
+		dockerStatus = info.DockerStatus{}
```

Comment:
Do you mean something like:

        dockerStatus := info.DockerStatus{}
        tempStatus, err := docker.Status()
        if err != nil {
                glog.Warningf("Unable to connect to Docker: %v", err)
        } else {
               dockerStatus = tempStatus
        }